Demo

ZhangZelin opened the pull request:
This is what we have from Game Jam 2

xAmaranth commented:
Hey Zelin,

A couple of notes:

Please scan through the files and ensure we're not bloating our assets folder or scene folder with assets or scenes that are not going to be used. For example, I don't believe we have any jigsaws, nor did we put any scenes under an ExampleScene folder.

I would also recommend renaming certain materials to be more descriptive. For the most part, I can guess what materials are used for what, but what is used where should be a matter of simple intuition, and not guesswork.

Also, I noticed we have a length camera movement script, though I only recall having used a camera script that caused it to look at the player. Is this a script we built?

ZhangZelin commented:
> Hey Zelin,
> 
> A couple of notes:
> 
> Please scan through the files and ensure we're not bloating our assets folder or scene folder with assets or scenes that are not going to be used. For example, I don't believe we have any jigsaws, nor did we put any scenes under an ExampleScene folder.
> 
> I would also recommend renaming certain materials to be more descriptive. For the most part, I can guess what materials are used for what, but what is used where should be a matter of simple intuition, and not guesswork.
> 
> Also, I noticed we have a length camera movement script, though I only recall having used a camera script that caused it to look at the player. Is this a script we built?

Hey Edison, I cleaned up the asset folder and renamed some of our material to be more intuitive.

As for the camera movement script, it was the a sample script that came with the LWRP example when I initialized the project as a LWRP project. Since we are not using it, I have deleted it.
